src/Compilers/CSharp/Portable/BoundTree/IBoundInvocableExpression.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal interface IBoundInvocableExpression
```

Comment:
>IBoundInvocableExpression [](start = 23, length = 25)

The name feels more specific than the content of the type. For example, given this contract, BoundField access can implement it too. Consider if we can achieve the goal without this interface completely, pass the base class around and extract information based on kind.

Reply:
I'll remove this.

---
In reply to: [236844876](https://github.com/dotnet/roslyn/pull/31010#discussion_r236844876) [](ancestors = 236844876)